Chinese/Demo_htx/demo.md

```diff
@@ -0,0 +1,3 @@
+click  to visit baidu
+
+click to visit raw url [go here](english/marshmallow.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.